app.rb

```diff
@@ -0,0 +1,128 @@
+# app.rb
+
+require_relative 'library'
+
+# General method to print a title
+def print_title(title)
+  puts "\n_____ #{title} _____\n\n"
+end
+
+# This class is responsible for the user interface and for
+# calling the methods of the Library class.
+class App
+  def initialize
+    @library = Library.new
+  end
+
+  def list_books(title = 'List of Books')
+    print_title('List of Books') unless title.empty?
+    books = @library.list_books
+    books.each.with_index { |book, index| puts "#{index + 1} -> #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_people(title = 'List of People')
+    print_title(title) unless title.empty?
+    people = @library.list_people
+    people.each.with_index { |person, index| puts "#{index + 1} -> #{person.name} (#{person.class.name})" }
+  end
```

Comment:
- [ ] Please ensure that when listing people, all their details are included exactly as shown in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).
  
  ![image](https://github.com/jicamargo/school_library/assets/73854999/8167ba1e-eca2-47cd-a0b7-6bc1bbc4c25f)

Reply:
✔️ Changes are done! Thank you 👍🏽

Reply:
✔️ Changes are done! Thank you 👍🏽